Assets/MRTK/Providers/WindowsMixedReality/XR2018/Controllers/WindowsMixedRealityArticulatedHand.cs

```diff
@@ -124,6 +129,8 @@ private void UpdateHandData(InteractionSourceState interactionSourceState)
                 return;
             }
 
+            Profiler.BeginSample("[MRTK] Windows Mixed Reality Articulate Hand: UpdateHandData");
```

Comment:
Also small-ish nit, but there seem to be a few formats these samples are using. For example, this one has spaces between each word of the class, but the one on line 98 is all together `WindowsMixedRealityArticulatedHand`.

Reply:
This might be the only example with spaces actually

Reply:
i tried a search/replace. i will fix the spaces